tests/invert_test.cpp

```diff
@@ -170,7 +169,7 @@ int main(int argc, char **argv)
     setQudaMgSolveTypes();
     setMultigridInvertParam(inv_param);
     // Set sub structures
-    mg_param.invert_param = &mg_inv_param;
+    mg_param.invert_param = &inv_param;
```

Comment:
This changes breaks the MG solver when run through invert_test.  This is primarily due to the poor interface of the multigrid solver, but it is what it is.  The reasoning is that the `QudaInvertParam` that is embedded in the `QudaMultigridParam` is used to drive the null space generation, and this is necessarily distinct from a solver that is preconditioned by MG.

Can you revert this change?